config/backend/component-config.local.json

```diff
@@ -0,0 +1,41 @@
+{
+    "loopback-component-explorer": {
+        "mountPath": "/explorer"
+    },
+    "loopback-component-visualizer": {
```

Comment:
I am not too sure this `visualizer` is needed for rabbitmq, can we remove it and keep it for later eventually?

Reply:
I admit that I don't fully understand this json config file because I haven't worked directly on the scicat backend. If you think it can be removed, I could remove this, test it and push an update to this PR.